src/Microsoft.AspNetCore.Blazor/Components/EventHandlers.cs

```diff
@@ -96,11 +99,14 @@ namespace Microsoft.AspNetCore.Blazor.Components
     [EventHandler("onvolumechange", typeof(UIEventArgs))]
     [EventHandler("onwaiting", typeof(UIEventArgs))]
 
-    // Error events
-    [EventHandler("onerror", typeof(UIErrorEventArgs))]
-
     // Progress events
+    [EventHandler("onloadstart", typeof(UIProgressEventArgs))]
+    [EventHandler("ontimeout", typeof(UIProgressEventArgs))]
+    [EventHandler("onabort", typeof(UIProgressEventArgs))]
+    [EventHandler("onload", typeof(UIProgressEventArgs))]
+    [EventHandler("onloadend", typeof(UIProgressEventArgs))]
     [EventHandler("onprogress", typeof(UIProgressEventArgs))]
+    [EventHandler("onerror", typeof(UIErrorEventArgs))]
```

Comment:
These additions are also defined below in the next section.. what am I missing here?

Reply:
Yes, that  next section is wrong. I think I mistakenly ctr+z on the delete line of the next section. Fixing.